tests/integration/frameworks/wordpress/test_wordpress_apply_filters.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+The agent should properly instrument Wordpress apply_filters hooks.
```

Comment:
```suggestion
The agent should properly instrument Wordpress apply_filters hooks and generate
`Framework/WordPress/Hook/<hook_name>` metrics for all detected hooks.
```

Reply:
However even the updated comment doesn't even give clear picture. WordPress instrumentation not only generates `Framework/WordPress/Hook/<hook_name>` metrics but also `Framework/WordPress/Plugin/<plugin_or_theme_name_that_hooks_onto_hook_name>` if the agent is able to convert `$callback` into `plugin_or_theme_name`. This metric will not be generated by this test because the agent is not able to convert `$callback` into `plugin_or_theme_name`.

Reply:
https://github.com/newrelic/newrelic-php-agent/pull/786/commits/407e6703538dd5fcb8af1013d75943304a212c8d demonstrates a different way wordpress hook system can be mocked where callback is installed for a given hook and then called at hook time.

Reply:
This will not be done to limit number of conflicts when syncing oapi branch.